Carousel/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Document</title>
+  <script src="https://kit.fontawesome.com/4ca0456cfc.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <div class="container">
+
```

Comment:
we don't add an empty line after ONLY opening a tag and before closing it

Reply:
fixed